pageserve/templates/index.html

```diff
@@ -33,8 +33,8 @@ <h2>test user auth</h2>
 <h2>Past posts</h2>
 {% for post in posts %}
 <p>Posted by {{post.author_id}} at {{post.created_at}}</p>
-
-<p>For event {{ post.event_id }}</p>
+<!-- TODO(cmei4444) - link to event info -->
+<p>For event with ID {{ post.event_id }}</p>
```

Comment:
Do you want to include this as a link instead?

Reply:
I wanted to include an option to open up details for a selected event on a separate page so someone could check that they were posting to the right event. 
It's not a super important feature so I was going to leave it to another PR if I had time.

Reply:
SGTM.